source/Octopus.Tentacle.CommonTestUtils/Builders/StartScriptCommandV3AlphaBuilder.cs

```diff
@@ -1,22 +1,24 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Octopus.Tentacle.Contracts;
+using Octopus.Tentacle.Contracts.Builders;
 using Octopus.Tentacle.Contracts.ScriptServiceV3Alpha;
 
-namespace Octopus.Tentacle.Contracts.Builders
+namespace Octopus.Tentacle.CommonTestUtils.Builders
 {
     public class StartScriptCommandV3AlphaBuilder
```

Comment:
Any change made here will have no impact on customers as no customer ever uses this builder. If that isn't correct this could be an issue as it is changing defaults.

Reply:
@LukeButters These builders were only used in Tentacle tests and in Server. In a subsequent PR that consumes these changes, we won't be using those builders in Server, hence why I moved them.

Reply:
I checked, these were moved into Contracts in #764 in Jan for use in DynamicWorkers, which will be updated to use the new contract type, so they can be moved back.

They are being moved in #855